wp_api/src/login/url_discovery.rs

```diff
@@ -606,33 +608,57 @@ pub(crate) fn construct_attempts(input_site_url: String) -> Vec<AutoDiscoveryAtt
     attempts
 }
 
-#[derive(Debug, Clone, thiserror::Error, uniffi::Error)]
+#[derive(Debug, Clone, thiserror::Error, uniffi::Error, WpDeriveLocalizable)]
 pub enum ParseApiRootUrlError {
-    #[error(
-        "Api root link header not found!\nStatus Code: '{:#?}'\nHeader Map: '{:#?}'",
-        status_code,
-        header_map
-    )]
     ApiRootLinkHeaderNotFound {
-        header_map: Arc<WpNetworkHeaderMap>,
         status_code: u16,
+        header_map: Arc<WpNetworkHeaderMap>,
     },
 }
 
-#[derive(Debug, thiserror::Error, uniffi::Error)]
+impl WpSupportsLocalization for ParseApiRootUrlError {
+    fn message_bundle(&self) -> wp_localization::MessageBundle {
+        match self {
+            ParseApiRootUrlError::ApiRootLinkHeaderNotFound {
+                status_code,
+                header_map,
+            } => WpMessages::api_root_link_header_not_found(
+                status_code.to_string(),
+                format!("{:#?}", header_map),
+            ),
+        }
+    }
+}
+
+#[derive(Debug, thiserror::Error, uniffi::Error, WpDeriveLocalizable)]
 pub enum FetchApiDetailsError {
-    #[error(
-        "Request execution failed!\nStatus Code: '{:?}'.\nResponse: '{}'",
-        status_code,
-        reason
-    )]
```

Comment:
It would be nice to implement the same syntax like `#[localized_error(...)]`, to replace the `impl WpSupportsLocalization for ...`